File: ClientApp/src/app/models/accounts/vendor.ts

// src/app/models/vendor.model.ts

export class Vendor {
  id: number;
  name: string;
  street: string;
  cityState: string;
  zip: string;
  phone: string;
  shortName: string;
  onWeb: 'Yes' | 'No';
}

File: Models/Accounts/Vendor.cs
namespace ProInternal.Models.Accounts
{
    public class Vendor
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Street { get; set; }
        public string CityState { get; set; }
        public string Zip { get; set; }
        public string Phone { get; set; }
        public string ShortName { get; set; }
        public string OnWeb { get; set; }  // "Yes"/"No"

    }

}
